# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acroread/install.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./INSTALL.orig	2008-06-25 18:16:52.000000000 +0200
+++ ./INSTALL	2008-09-03 09:01:03.000000000 +0200
@@ -571,6 +571,11 @@
     "$ReadInstallDir" \
     "$ReadPlatformTar" "$SearchPlatformTar" "$CustomPlatformTar"
 
+  echo "Adding T2 Patches ..."
+  for PF in *.diff; do
+    cat $PF | ( cd "$ReadInstallDir"; patch -p0)
+  done
+
   if [ "`uname -s`" = "SunOS" ] || [ "`uname -s`" = "Linux" ] || [ "`uname -s`" = "AIX" ] ; then
     MakeLibGnomeSpeechLinks "$ReadInstallDir" "$ReadConfig"
   fi
